lib/main.js

```diff
@@ -36,7 +36,7 @@ Lambda.prototype.setup = function () {
 };
 
 Lambda.prototype.run = function (program) {
-  this._createSampleFile('event.json');
+  this._createSampleFile('event.json', program.eventFile);
```

Comment:
I think it would be cleaner if we would just replace this by:
`this._createSampleFile(program.eventFile);` and disregard https://github.com/motdotla/node-lambda/pull/51/files#diff-16b7d75b70953cbf4755508170f2f24cR21 since the default would be `event.json` and if you overwrite it we should use that instead.

Thoughts?